apps/src/makerlab/BoardController.js

```diff
@@ -195,8 +202,12 @@ function initializeCircuitPlaygroundComponents() {
       freq: 100
     }),
 
-    gyro: new five.Gyro({
+    accelerometer: new five.Gyro({
```

Comment:
What's the purpose of the rename here, and/or is this not used anywhere yet?  Usually with a change like this I look for a corresponding usage change.